ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-ref.yaml

```diff
@@ -18,7 +18,7 @@ ref:
     - name: NODE_NAME
       default: ""
     - name: LABEL_SELECTOR
-      default: "node-role.kubernetes.io/master"
+      default: "node-role.kubernetes.io/control-plane"
```

Comment:
Hey @paigerube14, is the "node-role.kubernetes.io/control-plane" selector backwards compatible i.e will it work for our 4.17 test runs? Thoughts?

Reply:
The control plane label has been added for a while. I can find back to 4.15 in our runs

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-node-tests-415/1865245244052738048/artifacts/krkn-hub-node-tests-415/redhat-chaos-pod-network-chaos/build-log.txt

```
  labels:
    beta.kubernetes.io/arch: amd64
    beta.kubernetes.io/instance-type: m6a.xlarge
    beta.kubernetes.io/os: linux
    failure-domain.beta.kubernetes.io/region: us-west-2
    failure-domain.beta.kubernetes.io/zone: us-west-2b
    kubernetes.io/arch: amd64
    kubernetes.io/os: linux
    node-role.kubernetes.io/control-plane: ""
    node-role.kubernetes.io/master: ""
```

Reply:
Got it! Thanks!